isaaclab.bat

```diff
@@ -266,6 +257,8 @@ if "%arg%"=="-i" (
     )
     rem install the rl-frameworks specified
     !python_exe! -m pip install -e %ISAACLAB_PATH%\source\extensions\omni.isaac.lab_tasks[!framework_name!]
```

Comment:
I think we need to add `call !python_exe!...` here to allow execution to continue after this command. Otherwise it complains about "Invalid attempt to call batch label outside of batch script."

Reply:
Fixed this now.